src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -5324,6 +5321,7 @@ public class AddressSettings extends ClientSettings {
   }
 
 
+  @BetaApi("A restructuring of stub classes is planned, so this may break in the future")
```

Comment:
This is repeated in many places. Could/should it be a function?

Reply:
Hmm... I kind of like having this concrete instead of abstracted.